rgw/v2/lib/rgw_config_opts.py

```diff
@@ -116,7 +117,16 @@ class CephConfigSet:
     def set_to_ceph_cli(self, key, value):
         log.info("setting key and value using ceph config set cli")
         self.prefix = "sudo ceph config set"
-        self.who = "client.rgw"  # naming convention as ceph conf
+
+        cmd_ps = "ceph orch ps --daemon_type rgw -f json"
+        out_ps = utils.exec_shell_cmd(cmd_ps)
+        out = json.loads(out_ps)
+        service_name_list = []
+        for node in out:
+            service_name = node.get("service_name")
```

Comment:
Could you please use `daemon_name` instead of `service_name`.
`Service_name` will create new entry in `ceph config`

Reply:
@udaysk23 Thanks
I have done the changes

Reply:
Logs: http://magna002.ceph.redhat.com/cephci-jenkins/ameena/test_multisite_dynamic_resharding_greenfield.console.log